site/en/blog/extension-instantnav/index.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: 'layouts/blog-post.njk'
+title: "Chrome Extensions: Extending API to support Instant Navigation"
+description: >
+    The Extensions API has been updated to support back/forward cache, preloading navigations.
+authors:
+  - dtapuska
+hero: 'image/zKrSUSkPboWMTTSEkowJbqw5Egi2/r5IdmaTdE5wBS2d6CCQY.png'
+alt: >
+  Instant navigation in a tab.
+date: 2022-10-04
+---
+
+TL;DR: You may need to update your extension if it assumes `frameId == 0`
+is a page’s main frame. See below for details.
+
+Chrome has been working hard at making navigation fast. Instant Navigation
+technologies such as [Back/Forward Cache](https://web.dev/bfcache/)
+([shipped](https://chromestatus.com/feature/6279906713403392) on desktop in M96)
+and [Speculation Rules](https://chromestatus.com/feature/5740655424831488)
+(shipped in M103) improve both the going back and going forward experience.
+In this post we will explore the updates we’ve made to browser extensions APIs
+to accommodate these new workflows.
+
+## Understanding the Types of Pages
```

Comment:
```suggestion
## Understanding the types of pages
```

Reply:
This site uses sentence case for titles. Please fix this throughout. (I know it's not what your writing teachers taught you.)

Reply:
Done